dependencies/vaticle/repositories.bzl

```diff
@@ -21,7 +21,7 @@ def vaticle_dependencies():
     git_repository(
```

Comment:
This commit adds the three required libraries (ui-test-desktop, ui-test-junit4-desktop and kotlinx-coroutines-test) to vaticle_dependencies.